node_modules/oae-authentication/lib/strategies/facebook/init.js

```diff
@@ -65,12 +66,7 @@ module.exports = function() {
                 opts.email = profile.emails[0].value;
             }
 
-            if (opts.email) {
-                opts.emailVerified = true;
-            }
-
-            var context = new Context(tenant, null);
```

Comment:
We can get rid of the `var Context = require('oae-context');` line here and in a few similar files